app/services/home_service.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+# HomeService
+class HomeService
+  attr_reader :slider_books, :best_seller_books
+
+  def initialize
```

Comment:
No need for such services. It doesn't contain any logic, only assigns the instance variables. Please move this code back to controller